### DISCUSSION TITLE
文件存储：lotus clientStartDeal的wallet参数只能传lotus节点的wallet里账号？

### DISCUSSION THREAD

**Discussion by minnanlin88** [filecoin-project/community-china#137] [应用开发]
lotus clientStartDeal的wallet参数只能传lotus节点的wallet里账号？做dapp，如果用户是移动端，通过调用lotus的json rpc，这时候存储文件时，需要扣款的账号应该是移动端的用户的账号，此时，rpc对应的lotus里的wallet list又没有这个账号就会报错：provided address doesn't exist in wallet 这个情况要怎么解决？

**Comment by maxvint** [comment]
@minnanlin88  需要把发单的钱包地址，导入到rpc连接的这个daemon里面

**Reply by minnanlin88** [reply]
导入的话，我有考虑过，但是导入时需要终端用户的私钥，这样dapp终端的用户就把私钥给了这个lotus 节点，私钥泄露了（该节点就能随意获得用户的私钥），该如何解决？
